crates/sui-core/src/authority.rs

```diff
@@ -1019,6 +1025,7 @@ impl AuthorityState {
         checkpoints: Option<Arc<Mutex<CheckpointStore>>>,
         genesis: &Genesis,
         prometheus_registry: &prometheus::Registry,
+        tx_reconfigure_consensus: Sender<(ConsensusKeyPair, ConsensusCommittee)>,
```

Comment:
Just a comment, I see we are creating a new channel just for reconfiguration.  Are there potentially other control messages that we might wish to send to Narwhal in the future?  For example, change logging level, or some other thing like that?  I'm wondering if it makes sense to make this a general purpose control channel, or there is no such need?

Reply:
Good question. I do not know if we will need additional channels in the future (for now, reconfig is the only one I can think of)